resources/waf_rules.go

```diff
@@ -0,0 +1,64 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/waf"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func WafRules() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_waf_rules",
+		Resolver:     fetchWafRules,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Columns: []schema.Column{
+			{
+				Name:     "account_id",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSAccount,
+			},
+			{
+				Name:     "region",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSRegion,
+			},
+			{
+				Name: "name",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "rule_id",
+				Type: schema.TypeString,
```

Comment:
A rule has more data, just like you did in GetWebAcl you need to do the same for Rules and RuleGroups

Reply:
Thanks!